patches/chrome-browser-resources-settings-appearance_page-appearance_page.ts.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/settings/appearance_page/appearance_page.ts b/chrome/browser/resources/settings/appearance_page/appearance_page.ts
+index 2ef72f805303ebf050f011824b370d3f477a847b..76c82f929b51e040e4863e6b609066fd7cbdeccb 100644
+--- a/chrome/browser/resources/settings/appearance_page/appearance_page.ts
++++ b/chrome/browser/resources/settings/appearance_page/appearance_page.ts
+@@ -471,7 +471,7 @@ export class SettingsAppearancePageElement extends
+   }
+ 
+   private showHr_(previousIsVisible: boolean, nextIsVisible: boolean): boolean {
+-    return previousIsVisible && nextIsVisible;
++    return false && previousIsVisible && nextIsVisible;
```

Comment:
You're right. it didn't work because of new line. When I made them to one liner, my above code works.
However, query with wildcard doesn't work. it always gives zero array.
Tried `.hr[hidden*=showHr_]`, `.hr[hidden*=showHr_]`, `[hidden*="showHr_"]` but all don't work..

Reply:
@fallaciousreasoning Removed this patch also with 57998b37441df67dc8c48a13ca601c5757d7ea7e